packages/react-core/src/components/NotificationDrawer/NotificationDrawerGroup.tsx

```diff
@@ -32,28 +35,39 @@ export const NotificationDrawerGroup: React.FunctionComponent<NotificationDrawer
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   onExpand = (event: any, expanded: boolean) => undefined as any,
   title,
+  truncateTitle = 0,
   ...props
-}: NotificationDrawerGroupProps) => (
-  <section
-    {...props}
-    className={css(styles.notificationDrawerGroup, isExpanded && styles.modifiers.expanded, className)}
-  >
-    <h1>
-      <button
-        className={css(styles.notificationDrawerGroupToggle)}
-        aria-expanded={isExpanded}
-        onClick={e => onExpand(e, !isExpanded)}
-      >
-        <div>{title}</div>
-        <div className={css(styles.notificationDrawerGroupToggleCount)}>
-          <Badge isRead={isRead}>{count}</Badge>
-        </div>
-        <span className="pf-c-notification-drawer__group-toggle-icon">
-          <AngleRightIcon />
-        </span>
-      </button>
-    </h1>
-    {children}
-  </section>
-);
+}: NotificationDrawerGroupProps) => {
+  const titleRef = React.useRef(null);
+  React.useEffect(() => {
+    if (!titleRef.current || !truncateTitle) {
+      return;
+    }
+    titleRef.current.style.setProperty(maxLines.name, truncateTitle.toString());
+  }, [titleRef, truncateTitle]);
+
+  return (
+    <section
+      {...props}
+      className={css(styles.notificationDrawerGroup, isExpanded && styles.modifiers.expanded, className)}
+    >
+      <h1>
+        <button
+          className={css(styles.notificationDrawerGroupToggle)}
+          aria-expanded={isExpanded}
+          onClick={e => onExpand(e, !isExpanded)}
+        >
+          <div ref={titleRef}>{title}</div>
```

Comment:
I think this needs to be 

```suggestion
          <div ref={titleRef} className={css(styles.notificationDrawerGroupToggleTitle)}>{title}</div>
```